gauntlet/packages/gauntlet-solana-contracts/src/index.ts

```diff
@@ -5,25 +5,29 @@ import { io } from '@chainlink/gauntlet-core/dist/utils'
 import Solana from './commands'
 import { makeAbstractCommand } from './commands/abstract'
 import { defaultFlags } from './lib/args'
+import { SolanaCommand } from '@chainlink/gauntlet-solana'
+export { CONTRACT_LIST, getContract } from './lib/contracts'
 
-const commands = {
-  custom: [...Solana],
+export const commands = {
+  custom: [...Solana] as any[],
```

Comment:
Why is this? If it was giving an error it means some command is not compatible

Reply:
It was left while creating the multisig wrapper. I will remove it along with its comment below.